packages/cli/src/util/handle-error.ts

```diff
@@ -3,6 +3,44 @@ import info from './output/info';
 import errorOutput from './output/error';
 import { APIError } from './errors-ts';
 import { getCommandName } from './pkg-name';
+import { type Output } from './output/create-output';
+
+export function printError(error: unknown, output: Output) {
+  // Coerce Strings to Error instances
+  if (typeof error === 'string') {
+    error = new Error(error);
+  }
+
+  const apiError = error as APIError;
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** Can we remove the type assertion and use type checking instead?